python/ray/train/v2/_internal/execution/controller.py

```diff
@@ -111,16 +110,20 @@ def __init__(
             checkpoint_config=self._run_config.checkpoint_config,
             storage_context=self._storage_context,
         )
-
-        self._checkpoint_handler = CheckpointHandler(self._checkpoint_manager)
+        self._report_handler = ReportHandler(
```

Comment:
nit: Does this need to be an instance variable or should it just be a local variable? Seems unused outside of the init.
```suggestion
        report_handler = ReportHandler(
```